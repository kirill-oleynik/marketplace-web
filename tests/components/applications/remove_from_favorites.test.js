const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const {
  RemoveFromFavorites
} = require('../../../app/components/applications/remove_from_favorites');

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <RemoveFromFavorites
        t={() => {}}
        disabled
        onClick={() => {}}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
