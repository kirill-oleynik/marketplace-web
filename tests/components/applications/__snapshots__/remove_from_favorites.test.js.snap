// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<button
  className="favourites-activities"
  disabled={true}
  onClick={[Function]}
  type="button"
>
  <span
    className="favourites-activities__addeded"
  >
    <i
      className="icon icon-heart-filled favourites-heart__icon font-16 mr-10"
    />
    <span
      className="favourites-add__text"
    />
  </span>
  <span
    className="favourites-activities__remove"
  >
    <span
      className="relative mr-10"
    >
      <i
        className="icon icon-heart-filled favourites-heart__icon font-16"
      />
      <i
        className="icon icon-cross favourites-activities__remove-cross"
      />
    </span>
    <span
      className="favourites-add__text"
    />
  </span>
</button>
`;
